testsuite/features/secondary/srv_content_lifecycle.feature

```diff
@@ -34,26 +34,26 @@ Feature: Content lifecycle
     When I follow the left menu "Content Lifecycle > Projects"
     And I follow "clp_name"
     And I click on "Attach/Detach Sources"
-    And I select "SLES12-SP5-Pool for x86_64" from "selectedBaseChannel"
-    And I wait until I see "SLES12-SP5-Pool for x86_64" text
+    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "selectedBaseChannel"
+    And I exclude the recommended child channels
```

Comment:
Why? Don't we need them?

Reply:
If I am not mistaken for this feature we simply create a CLM project based on a based product, but we don't need to include every sub-channel - especially since we delete the project at the end of the feature without using it.
Not including them diminishes the amount of channels to cleanup at the end of the feature.